ONSdigital/ssdc-rm-toolbox
# remove sample loading, generation & validation

# Motivation and Context
We've got a shiny support tool.  We don't need toolbox to load or validate files anymore.  The generation code has been move to dev tools

# What has changed
remove sample load, validation and generation

# How to test?
Fire up toolbox and check that you can't load, validate or generate a sample anymore

(https://trello.com/c/xYVD4Kbo/3008-deprecate-the-toolbox-sample-validator-loader-code-3)
